From 37750c6de684320936b000032faa85728bf7f543 Mon Sep 17 00:00:00 2001
From: berbascum <berbascum@ticv.cat>
Date: Tue, 3 Sep 2024 22:02:51 +0200
Subject: [PATCH 3/5] (droidian) kernel-info: set 14 as clang version

---
 debian/kernel-info.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/kernel-info.mk b/debian/kernel-info.mk
index 640d67b1daf5..ee4bfe670c4c 100644
--- a/debian/kernel-info.mk
+++ b/debian/kernel-info.mk
@@ -214,8 +214,8 @@ ENABLE_VERBOSE = 0
 # Set clang version
 #CLANG_VERSION = 10.0-r370808
 #CLANG_VERSION = 11.0-r383902
-CLANG_VERSION = 12.0-r416183b
-#CLANG_VERSION = 14.0-r450784d
+#CLANG_VERSION = 12.0-r416183b
+CLANG_VERSION = 14.0-r450784d
 #CLANG_VERSION = 17.0-r487747
 
 # Set to 1 to use a manually installed toolchain
-- 
2.39.2

